src/Command/ConsumerCommand.php

```diff
@@ -91,7 +85,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $provider,
             $producer,
             $this->consolePath,
-            $input->getOption('env'),
+            $input->hasOption('env') ? $input->getOption('env') : null,
             true === $input->getOption('disable-verbosity-propagation') ? OutputInterface::VERBOSITY_QUIET : $output->getVerbosity()
```

Comment:
test also if `env` is not available. does not make sense if not disabling AND not having env.